bin/improver-tests

```diff
@@ -170,7 +184,13 @@ function improver_test_cli {
     if [[ -n $DEBUG_OPT ]]; then
         VERBOSE_OPT='-v'
     fi
-    pytest -m acc ${VERBOSE_OPT:-}
+
+    if [[ ! ${IMPROVER_IGNORE_CHECKSUMS:-} = "true" ]]; then
+      pytest $CLISUBTEST/acceptance/test_checksums.py -k test_kgo_checksums
```

Comment:
```suggestion
      pytest $CLISUBTEST/acceptance/test_checksums.py
```
It'd be nice to use markers here as well, but then the full pytest discovery would have to run twice. I think specifying the test python code path is flexible enough and will avoid the full directory scan for this small test run.

Reply:
Done.